gwu-libraries/sfm-ui
# T141 Implements limited apt package wildcarding

To test:

Temporarily substitute the URL
`https://raw.githubusercontent.com/gwu-libraries/sfm-ui/t141-package-wildcarding/requirements/requirements.apt` within the `ADD` directive near the top of your Dockerfile which gets `requirements.apt` and places it within the container.

Build in a completely clean environment (meaning, remove all docker images first).

If successful, you should not see errors like the error described in #164.
